config/src/main/java/no/difi/move/common/config/KeystoreProperties.java

```diff
@@ -1,21 +1,23 @@
 package no.difi.move.common.config;
 
 import lombok.Data;
+import lombok.ToString;
 import org.springframework.core.io.Resource;
 
 import java.security.KeyStore;
 
-/**
- *
- * @author Nikolai Luthman <nikolai dot luthman at inmeta dot no>
- */
 @Data
+@ToString(exclude = "password")
```

Comment:
Exclude property her er @Deprectated i nye versjoner av Lombok. Anbefaler å bruke  @ToString.Exclude på propertiene du ønsker å eksludere. TIlsvarende for @EqualsAndHashCode.Exclude